### PR TITLE
Fix CSS Problem for build and deployment.

### PR DESCRIPTION
**Pull request by MrPrimate**

### CODE REVIEW

File: src/activeLighting.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ class ATL {
             if (!gm) return;
             let linkedTokens = canvas.tokens.placeables.filter(t => !t.document.link)
             for (let token of linkedTokens) {
-                let ATLeffects = token.actor.effects.filter(entity => !!entity.changes.find(effect => effect.key.includes("ATL")))
+                let ATLeffects = token.actor?.effects?.filter(entity => !!entity.changes.find(effect => effect.key.includes("ATL")))
                 if (ATLeffects.length > 0) ATL.applyEffects(token.actor, ATLeffects)
```
Comment (kaelad02) [inline src/activeLighting.js]: Wouldn't this also need to be updated with either a `ATLeffects &&` or `ATLeffects?.`? If `token.actor` or something else in the chain was `undefined` then `ATLeffects` would be `undefined` as well and we'd still get an error

File: src/activeLighting.js [hunk 2]
```diff
@@ -141,7 +141,7 @@ class ATL {
         if (!gm) return;
         let linkedTokens = canvas.tokens.placeables.filter(t => !t.document.link)
         for (let token of linkedTokens) {
-            let ATLeffects = token.actor.effects.filter(entity => !!entity.changes.find(effect => effect.key.includes("ATL")))
+            let ATLeffects = token.actor?.effects?.filter(entity => !!entity.changes.find(effect => effect.key.includes("ATL")))
             if (ATLeffects.length > 0) ATL.applyEffects(token.actor, ATLeffects)
```
Comment (kaelad02) [inline src/activeLighting.js]: same thing here

File: module.json [hunk 3]
```diff
@@ -1,34 +1,31 @@
...
-	"bugs": "https://github.com/kandashi/Active-Token-Lighting/issues"
+  "id": "ATL",
+  "title": "Active Token Effects",
+  "description": "Active Token Effects",
+  "authors": [
+    {
+      "name": "Kandashi"
+    }
+  ],
+  "version": "0.4.02",
```
Comment (kaelad02) [inline module.json]: I'm a little nervous pulling in a version number change. But it does release with the GitHub Action so it should be fine, right?

File: src/activeLighting.js (hunk 1 above)
Comment (MrPrimate) [reply src/activeLighting.js]: Yes, these were from someone else's PR, but you're quite right

File: module.json (hunk 3 above)
Comment (MrPrimate) [reply module.json]: This is fine, the version is pulled from the tag/release by the action

**Comment by MrPrimate** [comment]
Hopefully I have addressed these issue in the last updates

**Pull request merged**